setup.py

```diff
@@ -28,7 +28,6 @@ def find_version(*file_paths):
 
     install_requires = [
         'requests>=2.7,<3.0',
-        'requests-kerberos',
```

Comment:
move it to `extras_require`

```
    extras_require={
        'kerberos':  ['requests-kerberos'],
    },
```

to install with extra requirements, specify dependency with:

```
pip install yarn-api-client[kerberos]
```

Reply:
Great, I didn't know about this, much more flexible